libs/protocol-client/src/lib/client/custom.ts

```diff
@@ -16,6 +16,12 @@ export class Custom extends BaseClient {
   }
 
   public async createUserContribution(args: UserContributionCreateInput) {
+    await this.sdk.getOrCreateActivityType({
```

Comment:
Should we get the response of this function call and update the args for createUserContribution?